help/en/html/doc/Technical/JUnit.shtml

```diff
@@ -189,6 +189,12 @@
             <dd>When set true, run some extra tests; usually used during code migration,
                 where not everything is right yet but you want to be able to include
                 tests for individual running.  </dd>
+        <dt>jmri.jmrit.audio.DefaultAudioManager.implementation</dt>
+            <dd>When set, use the specified class for the AudioFactory implementation.
+                Setting it to <code>jmri.jmrit.audio.NullAudioFactory</code> will 
```

Comment:
This section is about more than just enable/disable some tests. Maybe the name of this section and the description on lines 172 - 178 could be changed or improved. For example, `jmri.shutdownmanager` is used in our standard testing infrastructure(s) to ensure that a mocked test-compatible ShutDownManager is available.

Maybe the section should be split in two sections, "Optional tests" and "Configuration of tests".

Reply:
With "this section", I mean the part starting at line 172 and forward.

Reply:
Did a bit of reorg there, let me know if I understood what you're looking for.

Reply:
Thanks.

I suggest also move `jmri.log4jconfigfilename`, `jmri.demo` and `jmri.util.JUnitUtil.printSetUpTearDownNames` to the new section  `Controlling Test Operation`.

Reply:
Done. Thanks!